File: 1_easy/03679_compute_decimal_representation/solution.py
class Solution:
    def decimalRepresentation(self, n: int) -> List[int]:
        multiplier = 1
        res = []
        while n > 0:
            tmp = (n % 10) * multiplier;
            if tmp != 0:
                res.append(tmp)
            multiplier *= 10
            n //= 10

        res.reverse()

        return res

File: 1_easy/03679_compute_decimal_representation/solution.rs
impl Solution {
    pub fn decimal_representation(mut n: i32) -> Vec<i32> {

        let mut multiplier = 1;
        let mut res: Vec<i32> = Vec::new();

        while n > 0 {
            let tmp = (n % 10) * multiplier;

            if tmp != 0 {
                res.push(tmp);
            }

            multiplier *= 10;
            n /= 10;
        }

        res.reverse();

        res
    }
}
